CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
 
 
+## 18.0.2 - 2018-11-01
+Updated:
+- styles and padding of Checkbox and RadioGroup to match some changes to the BDA UI Toolkit
+- styleguidist example page updated to be more responsive
+
+## 18.0.1 - 2018-11-01
+Fixed:
+- minor tabPanel change to select active tab based on key
+
```

Comment:
With the recent changes I made you shouldn't need to push the updated changelog and version bumps anymore.